### PR TITLE
feat: Added syntax highlighting

### LINKED ISSUE #49: Add Code highlight for Casbin CSV policy
e.g., https://casbin.org/docs/abac

image

The code coloring should be like: https://casbin.org/editor (source code is: https://github.com/casbin/casbin-editor)

image

### PR DESCRIPTION
**Pull request by EmperorYP7**
Part for: #49 

### Description

- Added syntax highlighting for various languages including `ini` and `csv`

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -148,6 +148,7 @@ module.exports = {
     prism: {
       theme: lightCodeTheme,
       darkTheme: darkCodeTheme,
+      additionalLanguages: ['ini', 'csv', 'lua', 'csharp', 'cpp', 'java', 'php', 'rust', 'json', 'bash'],
```
Comment (Selflocking) [inline docusaurus.config.js:151]:
```suggestion
      additionalLanguages: ['ini', 'csv', 'lua', 'csharp', 'java', 'php', 'rust'],
```

refer to https://docusaurus.io/docs/markdown-features/code-blocks#supported-languages
`cpp`, `json`,`bash` are support by default.

File: docs/ABAC.mdx [hunk 2]
```diff
@@ -9,7 +9,7 @@ ABAC is ``Attribute-Based Access Control``, meaning you can use the attributes (
 
 Use the official ABAC example for example:
 
-```ini
+```ini title="model.ini"
```
Comment (hsluoyz) [inline docs/ABAC.mdx:12]: Why add title? We need to systematically decide why, when and where to add title or not

File: docs/GetStarted.mdx [hunk 3]
```diff
@@ -366,7 +366,7 @@ if (e.enforce(sub, obj, act) == true) {
 
 <TabItem value="Node.js" label="Node.js">
 
-```js
+```javascript
```
Comment (hsluoyz) [inline docs/GetStarted.mdx:369]: What's the difference between `js` and `javascript`?

File: docs/Tutorial.mdx [hunk 4]
```diff
@@ -14,7 +14,7 @@ Before reading, please note that some tutorials are for the Casbin's model and w
 
 This paper digs deeply into the design details about Casbin. Please cite the following BibTex if you use Casbin/PML as a reference in your paper:
 
-```bibtex
+```latex
 @article{luo2019pml,
   title={PML: An Interpreter-Based Access Control Policy Language for Web Services},
   author={Luo, Yang and Shen, Qingni and Wu, Zhonghai},
```
Comment (Selflocking) [inline docs/Tutorial.mdx:20]: Maybe it's not `latex`.

If it is indeed `latex`, we need add `latex` in `docusaurus.config.js`.

File: blog/2021-08-19-apisix-casbin-authorization.md [hunk 5]
```diff
@@ -47,7 +47,7 @@ m = (g(r.sub, p.sub) || keyMatch(r.sub, p.sub)) && keyMatch(r.obj, p.obj) && key
 
 From the above scenario, the policy would be:
 
-```conf
+```csv
```
Comment (hsluoyz) [inline blog/2021-08-19-apisix-casbin-authorization.md:50]: The `csv` tag doesn't give colorzing. Is there any way to improve the format for CSV policy? Like another tag or even write our own formatting plugin for a new tag.

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: docs/ABAC.mdx (hunk 2 above)
Comment (EmperorYP7) [reply docs/ABAC.mdx]: We might tackle this in another PR

File: docs/GetStarted.mdx (hunk 3 above)
Comment (EmperorYP7) [reply docs/GetStarted.mdx]: nothing. Was testing something, went through into the commit.

File: blog/2021-08-19-apisix-casbin-authorization.md (hunk 5 above)
Comment (EmperorYP7) [reply blog/2021-08-19-apisix-casbin-authorization.md:50]: This is a problem. Prism plugin for `csv` is not yielding the results we want. I guess this calls for some custom code.

**Review by Selflocking — APPROVED** [review]

File: blog/2021-08-19-apisix-casbin-authorization.md (hunk 5 above)
Comment (hsluoyz) [reply blog/2021-08-19-apisix-casbin-authorization.md:50]: OK

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**